### PR TITLE
fix(l10n): fissure and invasion node key

### PR DESCRIPTION
**Pull request by SlayerOrnstein**

### CODE REVIEW

File: lib/Invasion.js [hunk 1]
```diff
@@ -52,7 +52,13 @@ class Invasion extends WorldstateObject {
...
      * @type {string}
      */
-    this.node = translator.node(data.Node, locale);
+    this.nodeKey = translator.node(data.Node, locale);
+
+    /**
+    * The node key where the invasion is taking place
+    * @type {string}
+    */
+    this.node = translator.node(data.Node);
```
Comment (TobiTenno) [inline lib/Invasion.js:61]: i think you swapped these
```suggestion
    this.node = translator.node(data.Node, locale);

    /**
    * The node key where the invasion is taking place
    * @type {string}
    */
    this.nodeKey = translator.node(data.Node);
```

**Review by TobiTenno — CHANGES_REQUESTED** [review]

**Comment by TobiTenno** [comment]
i'm gonna grab your fork and drop the extra parts for pr's since apparently lint doesn't work on fork :-/

**Comment by TobiTenno** [comment]
Or I won't, guess I don't have access on your fork to push to this branch.

**Comment by SlayerOrnstein** [comment]
That's alright

**Comment by TobiTenno** [comment]
Yeah, but i need to make sure this doesn't break anything downstream in the tests, lol

**Comment by SlayerOrnstein** [comment]
Lol yeah I get it don't worry

**Comment by TobiTenno** [comment]
Could you possibly move your branch to the org repo and submit from that?

**Comment by SlayerOrnstein** [comment]
@TobiTenno there we go

**Pull request merged**